packages/webapp/src/utils/__tests__/credits-application.spec.js

```diff
@@ -7,11 +7,17 @@ import applicant from '../../__mocks__/applicant'
 describe('credits-application', () => {
   it('Should process typical application based on test data including LPA code', () => {
     const session = setCreditsApplicationSession()
+
+    session.set(creditsPurchaseConstants.redisKeys.CREDITS_PURCHASE_APPLICATION_REFERENCE, 'BNGCRD-L4XCQ-AIZMO')
```

Comment:
This should be `CREDITS_PURCHASE_PLANNING_APPLICATION_REF` for the planning application reference